src/main/java/net/dries007/tfc/client/overworld/LevelRendererExtension.java

```diff
@@ -419,7 +431,28 @@ public boolean renderSnowAndRain(ClientLevel level, int ticks, float partialTick
                                 }
 
                                 stateFlag = 0;
-                                RenderSystem.setShaderTexture(0, RAIN_LOCATION);
+
+
+                                // select the texture based on the amount of rain
```

Comment:
I'd rather these be in an array with a `RAIN_LOCATIONS[(int) (rainIntensity * 4)]` as it's one line of code vs. several